Formula/nethack.rb

```diff
@@ -56,4 +56,8 @@ def install
     chmod "g+w", libexec
     chmod "g+w", libexec+"save"
   end
+
+  test do
+    assert_match "NetHack, Copyright 1985-2015", shell_output("#{bin}/nethack", "q")
```

Comment:
The second argument to `shell_output` is supposed to be the exit code if non-zero.